src/app/(mobile-ui)/withdraw/manteca/page.tsx

```
@@ -0,0 +1,587 @@
+    }, [selectedMethodType])
+
+    const validateDestinationAddress = async (value: string) => {
+        value = value.trim()
+        if (!value) {
+            return false
+        }
+
+        let isValid = false
+        switch (countryPath) {
+            case 'argentina':
+                const { valid, message } = validateCbuCvuAlias(value)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)